cypress/e2e/views/archetype.view.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+export const archetypeName = "#name";
+export const description = "#description";
+export const criteriaTags = "input[aria-label='criteria-tags-select-toggle']";
+export const archetypeTags = "input[aria-label='archetype-tags-select-toggle']";
+export const stakeholder = "input[aria-label='stakeholder-select-toggle']";
+export const stakeholderGroups = "input[aria-label='stakeholder-groups-select-toggle']";
```

Comment:
We should avoid using aria-labels, please open a ticket to add reasonable selectors if there is no other way to get those objects

Reply:
I have also used aria-labels in a few places when there is no other way to get the objects.